integrationtest/cdi/pom.xml

```diff
@@ -0,0 +1,235 @@
+<!--
+
+     SPDX-License-Identifier: Apache-2.0
+
+     Copyright The original authors
+
+     Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.moditect</groupId>
+    <artifactId>moditect-integrationtest-parent</artifactId>
+    <version>1.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>moditect-integrationtest-cdi</artifactId>
+  <name>ModiTect Integration Test - CDI SE example</name>
+
+  <properties>
+    <weld.version>5.1.2.Final</weld.version>
+    <jakartaee-web-api.version>10.0.0</jakartaee-web-api.version>
+    <jboss-logging.version>3.5.3.Final</jboss-logging.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.weld.se</groupId>
+      <artifactId>weld-se-core</artifactId>
+      <version>${weld.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.platform</groupId>
+      <artifactId>jakarta.jakartaee-web-api</artifactId>
+      <version>${jakartaee-web-api.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging</artifactId>
+      <version>${jboss-logging.version}</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.12.1</version>
```

Comment:
We probably should use latest which is `3.13.0`. Please define a property in the parent pom that overrides this value https://github.com/moditect/moditect-org-parent/blob/b825070b22eec75ae8f965dfd7fd5e01898fc8bf/pom.xml#L56